design/high-level-reactivity.md

```diff
@@ -151,12 +184,22 @@ The rest of the document talks in terms of polling a dependency server for updat
 1. The invalidation server could accept websocket connections, and let the client subscribe to the dependency keys it cares about - this would mean the invalidation is pushed immediately, and then there is one roundtrip to fetch the actual data.
 2. The application server could subscribe to the invalidations, and do the GraphQL queries _on the server_, and then diff that against the current state of the client and send a patch. This would make the system work almost exactly like Meteor today, and could be a good option for applications with fewer users and a great need for low latency.
 
+// I think it's better to go with Just the 1. We could optimize on top of that. It's not a good idea to maintain two different things.
```

Comment:
I just want to emphasize that this design doesn't _stop_ you from implementing 2. I agree we shouldn't have it in the core implementation.